src/test/warnTooManyClasses.test.js

```diff
@@ -8,8 +8,8 @@ import styleSheet from '../models/StyleSheet'
 let styled
 
 describe('warn too many classes', () => {
-  const nativeWarn = console.warn
```

Comment:
Hmm why rename this? It was correctly described before, meaning native console.warn vs a mocked console.

Reply:
Can revert this, it's a leftover actually.